input/data/2013/2013ApJ...779...88A/tev-000014-lc.ecsv

```diff
@@ -0,0 +1,29 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: flux, unit: cm-2 s-1 TeV-1, datatype: float64}
```

Comment:
@GernotMaier - I'm doing a final review of this PR.

Here in `input/data/2013/2013ApJ...779...88A/tev-000014-lc.ecsv` you have a column `flux` which we use for integral fluxes, but a unit of `cm-2 s-1 TeV-1` which is for differential fluxes `dnde`. Please fix.